AUTHORS.txt

```diff
@@ -56,4 +56,5 @@ The following developers contributed to gcovr (ordered alphabetically):
     Will Thompson,
     William Hart,
     Antonio Quarta,
+    Zachary J. Fields
```

Comment:
Definitely authors list is not in the alphabetical order :).

Reply:
I'll sort the list.